software/util/qr_code_generator.py

```diff
@@ -0,0 +1,18 @@
+import pyqrcode
+import png
+
+
```

Comment:
Changed in #2ae18f7e4b3e535afc67c2bad1946bdd18b2dd91, not sure why it doesn't show up. Must be the new github review thing.